styleguide/source/assets/js/alert.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file
+ * alert.
+ *
+ * JavaScript should be made compatible with libraries other than jQuery by
+ * wrapping it with an "anonymous closure". See:
+ * - https://drupal.org/node/1446420
+ * - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
+ */
+(function ($) {
+  $(document).ready(function() {
+    console.log('doc ready');
+
+    // If the 'hide cookie is not set we show the alert
+    if (!readCookie('hideAlert')) {
+      console.log('no cookie');
+      $('.ama__alert').fadeIn("slow");
+    }
+
+    // Add the event that closes the popup and sets the cookie that tells us to
+    // not show it again until one day has passed.
+    $('.ama__alert__close').click(function() {
+      $('.ama__alert').fadeOut();
+      console.log('cookie created');
+      createCookie('hideAlert', true, 1);
+      return false;
+    });
+
+  });
+
+  // ---
+  // And some generic cookie logic
+  // ---
+  function createCookie(name,value,days) {
```

Comment:
I think we have the jQuery cookie already being used somewhere else. It should be used so we don't double up on libraries or functionality. 
https://github.com/js-cookie/js-cookie